python/hopsworks/core/environment_api.py

```diff
@@ -99,30 +110,37 @@ def get_environment(self):
 
         env_api = project.get_environment_api()
 
-        env = env_api.get_environment()
+        env = env_api.get_environment("my_custom_environment")
 
         ```
         # Returns
             `Environment`: The Environment object
         # Raises
             `RestAPIError`: If unable to get the environment
         """
-        project_envs = self._get_environments()
-        if len(project_envs) == 0:
-            return None
-        elif len(project_envs) > 0:
-            return project_envs[0]
-
-    def _delete(self, python_version):
-        """Delete the project Python environment"""
+        _client = client.get_instance()
+
+        path_params = ["project", self._project_id, "python", "environments", name]
+        query_params = {"expand": ["libraries", "commands"]}
+        headers = {"content-type": "application/json"}
+        return environment.Environment.from_response_json(
+            _client._send_request(
+                "GET", path_params, query_params=query_params, headers=headers
+            ),
+            self._project_id,
+            self._project_name,
+        )
+
+    def _delete(self, name):
```

Comment:
Is this method "not exposed" on purpose? or should it be `def delete(self, name)` without underscore?
Can you add `#Arguments` with the description of the name parameter?

Reply:
Yes the _delete is hidden by design. The delete operation is done on the environment object itself.